vnext/Microsoft.ReactNative/ReactNativeAppBuilder.idl

```diff
@@ -0,0 +1,43 @@
+
+import "ReactNativeHost.idl";
+import "ReactInstanceSettings.idl";
+import "IReactPackageProvider.idl";
+
+#include "DocString.h"
+
+namespace Microsoft.ReactNative
+{
+    [experimental]
+    DOC_STRING("This is the builder for creating ReactInstanceSettings.")
+    runtimeclass ReactInstanceSettingsBuilder
```

Comment:
Does this class add much value?  Seems like it would quickly become basically the same as the existing ReactInstanceSettings.

Reply:
Yeah I believe it does. It is more of a builder for ReactInstanceSettings. It makes it easy to setup ReactInstanceSettings. Existing ReactInstanceSettings is much more than what this builder is doing. This builder's responsibility is to help setup the settings in one go.